src/SAWScript/Interpreter.hs

```diff
@@ -1871,29 +1883,34 @@ primitives = Map.fromList
     , ""
     , "For example,"
     , ""
-    , "  r <- sat abc <thm>"
+    , "  r <- sat abc <prop>"
     , "  caseSatResult r <unsat> <sat>"
     , ""
-    , "will run '<unsat>' if '<thm>' is unSAT and will run '<sat> <example>'"
-    , "if '<thm>' is SAT, where '<example>' is a satisfying assignment."
-    , "If '<thm>' is a curried function, then '<example>' will be a tuple."
+    , "will run '<unsat>' if '<prop>' is unSAT and will run '<sat> <example>'"
+    , "if '<prop>' is SAT, where '<example>' is a satisfying assignment."
+    , "If '<prop>' is a curried function, then '<example>' will be a tuple."
+    , "If we could not determine the satisfiability of '<prop>', then"
+    , "this will run '<sat> {{ () }}'."
     ]
 
-  , prim "caseProofResult"     "{b} ProofResult -> b -> (Term -> b) -> b"
+  , prim "caseProofResult"     "{b} ProofResult -> (Theorem -> b) -> (Term -> b) -> b"
     (\_ _ -> toValueCase caseProofResultPrim)
     Current
     [ "Branch on the result of proving."
     , ""
-    , "Usage: caseProofResult <code to run if true> <code to run if false>."
+    , "Usage: caseProofResult <result> <code to run if true> <code to run if false>."
     , ""
     , "For example,"
     , ""
-    , "  r <- prove abc <thm>"
+    , "  r <- prove abc <prop>"
     , "  caseProofResult r <true> <false>"
     , ""
-    , "will run '<true>' if '<thm>' is proved and will run '<false> <example>'"
-    , "if '<thm>' is false, where '<example>' is a counter example."
-    , "If '<thm>' is a curried function, then '<example>' will be a tuple."
+    , "will run '<true> <thm>' if '<prop>' is proved (where '<thm>' represents"
+    , "the proved theorem) and will run '<false> <example>'"
+    , "if '<prop>' is false, where '<example>' is a counter example."
+    , "If '<prop>' is a curried function, then '<example>' will be a tuple."
+    , "If the proof of <prop> was not finished, but we did not find a counterexample,"
+    , "the example will run '<false> {{ () }}'"
```

Comment:
This will likely lead to a type error in an argument designed to process a counter-example, but perhaps that's fine.

Reply:
Yeah, I wasn't entirely sure what else to do here.